pkg/terraform/exec/plugins/Gopkg.lock

```diff
@@ -441,6 +552,55 @@
   revision = "51d6538a90f86fe93ac480b35f37b2be17fef232"
   version = "v2.2.2"
 
+[[projects]]
```

Comment:
Ooo, this does not look right, how is pulling in k8s.io/apimachinery ?

Reply:
I dunno.  I'll regenerate tonight with the debugging cranked up to see.